packages/components/src/components/remaining-time/remaining-time.tsx

```diff
@@ -5,13 +5,13 @@ import moment from 'moment';
 type TRemainingTimeProps = {
     end_time?: number;
     start_time: moment.Moment;
-    format: string;
+    format?: string;
     getCardLabels: () => { [key: string]: string }; // TODO Use the one from common after contract-card migration
 };
 
 const RemainingTime = ({ end_time, format, getCardLabels, start_time }: TRemainingTimeProps) => {
     if (!end_time || start_time.unix() > +end_time) {
-        return '';
+        return <React.Fragment />;
```

Comment:
@jim-deriv could you please check if there a chance it might affect the functionality? is the app working correctly when launched with this code? we had an issue with some of components where children where wrapped into <React.Fragment /> during TS migration which led to the UI breaking

Reply:
@maryia-binary,

It doesn't break the part it's used on as seen in the screenshot, however, I'll just make it as close as possible to what was there just incase it's used in any other place. I'll use this instead `<React.Fragment>{''}</React.Fragment>` .

![remaining time 2](https://user-images.githubusercontent.com/104334373/201609165-9bc7c5c1-10b0-4810-9395-37ef5daa7705.png)
![remaining time](https://user-images.githubusercontent.com/104334373/201609184-b0adeb78-abb8-4317-b4db-8d780b268f9e.png)